Makefile

```diff
@@ -23,6 +23,8 @@ CFLAGS_EMCC+=-s INVOKE_RUN=0
 CFLAGS_EMCC+=-s ALLOW_MEMORY_GROWTH=1
 CFLAGS_EMCC+=-s ALLOW_TABLE_GROWTH=1
 CFLAGS_EMCC+=-s FILESYSTEM=0
+CFLAGS_EMCC+=-s RESERVED_FUNCTION_POINTERS=1024
+CFLAGS_EMCC+=-s BINARYEN_TRAP_MODE='clamp'
```

Comment:
why is this added?

Reply:
update: as of #15, this project officially uses emcc 1.39.18 - I don't think BINARYEN_TRAP_MODE is supported on the newer emcc version. emscripten setup is now documented in README.md, and the build is tested in CI.

Reply:
I had problems loading a large source file without it - it might have been my older emcc version.
I'll open a separate PR on this if the issue persists
Reverted both changes